serious_stuff/lib/components/Fab.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_speed_dial/flutter_speed_dial.dart';
+import 'package:serious_stuff/constants.dart' as Constants;
+
+SpeedDial buildOptionsFAB() {
+  // The options floating action button.
+  return SpeedDial(
+    marginRight: 18,
+    marginBottom: 20,
+    animatedIcon: AnimatedIcons.menu_close,
+    animatedIconTheme: IconThemeData(size: 22.0),
+    visible: true,
+    closeManually: false,
+    curve: Curves.bounceIn,
+    overlayColor: Colors.black,
+    overlayOpacity: 0.5,
+    onOpen: () => print('OPENING DIAL'),
+    onClose: () => print('DIAL CLOSED'),
+    tooltip: 'Options',
+    heroTag: 'options-fab',
+    backgroundColor: Colors.white,
+    foregroundColor: Colors.black,
```

Comment:
Is the "backgroundColor" and "foregroundColor" related to button colors? Because I remember you mentioned you wanted global variables for theme colors. It's not something necessary to change rn, we can just note this down in Trello/Google Docs so that when we do get those global vars, we'll know what to change

Reply:
I actually created a `constants.dart` file to keep all the constants: `serious_stuff/lib/constants.dart`, and I used it for the speed dial children as well. Forgot to change it here. Hehe. Good catch.